src/TreeContainer.js

```diff
@@ -294,6 +298,7 @@ export const AugmentedTreeContainer = connect(
         _dashprivate_loadingState: getLoadingState(ownProps._dashprivate_layout, stateProps.requestQueue),
         _dashprivate_paths: stateProps.paths,
         _dashprivate_requestQueue: stateProps.requestQueue,
+        config: stateProps.config,
```

Comment:
Please rename as `_dashprivate_config` -- since these props are public / accessible from custom components, and yet purely internal details, the only protection we have from people using them (and they have) is by convention, prepending the name so it's obvious they are doing so at their own risk.